### PR TITLE
doc: update development.md, maintainers.md and security.md

### PR DESCRIPTION
**Pull request by pedaars**
Development.md

- change the path to the `AGENT_PROTO_COMPATIBILITY_MINIMUM_VERSION` and added a link to the const.ts file.
- add white space to improve readability

Maintainers.md

- update grammar to improve readability

Security.md

- add links to dyrector.io
- add mailto links to email address

### CODE REVIEW

**Review by polaroi8d — CHANGES_REQUESTED** [review]
Please adhere to the CONTRIBUTING.md guidelines for the PR title. Otherwise, great job! Thank you!

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -5,17 +5,20 @@ The internal details of dyrector.io's components.
 ## Agent versioning
 
 Crux considers two things to determine if an agent is outdated:
- - The `AGENT_PROTO_COMPATIBILITY_MINIMUM_VERSION` in the `web/crux/shared/const.ts` file
- - And the version number in Crux's `package.json`
+
+-   The `AGENT_PROTO_COMPATIBILITY_MINIMUM_VERSION` in the [web/crux/src/const.ts](./web/crux/src/shared/const.ts) file
```
[the remark this answers is not in the document]
Comment (pedaars) [reply DEVELOPMENT.md]: matches the other files

[the remark this answers is not in the document]
Comment (polaroi8d) [reply DEVELOPMENT.md:9]: Thank you, LGTM for my side. Wait at least one more approval and we can merge it.

**Review by polaroi8d — APPROVED** [review]

**Comment by gerimate** [comment]
LGTM, good job!

**Review by gerimate — APPROVED** [review]

**Review by robot9706 — APPROVED** [review]

**Pull request merged**